assets/js/modules/adsense/index.js

```diff
@@ -113,10 +114,26 @@ export const registerWidgets = ( widgets ) => {
 			priority: 1,
 			wrapWidget: false,
 			modules: [ 'adsense', 'analytics-4' ],
-			isActive: ( select ) =>
-				select( CORE_USER ).isKeyMetricActive(
-					KM_ANALYTICS_ADSENSE_TOP_EARNING_CONTENT
-				),
+			isActive: ( select ) => {
+				const isViewOnly = ! select( CORE_USER ).isAuthenticated();
+
+				if (
+					! select( CORE_USER ).isKeyMetricActive(
+						KM_ANALYTICS_ADSENSE_TOP_EARNING_CONTENT
+					)
+				) {
+					return false;
+				}
+
+				const isAdSenseLinked =
+					select( MODULES_ANALYTICS_4 ).getAdsenseLinked();
```

Comment:
@benbowler this also needs an update to use the correct case:

```suggestion
					select( MODULES_ANALYTICS_4 ).getAdSenseLinked();
```